pkg/k8sutils/k8sutils.go

```diff
@@ -45,6 +45,12 @@ const (
 	ObjectLockDefaultIncrementalCount = 5
 	//minProtectionPeriod defines minimum number of days, the backup are protected via object-lock feature
 	minProtectionPeriod = 1
+	// RestoreVolumeBatchCountKey - restore volume batch count value
+	RestoreVolumeBatchCountKey = "restore-volume-backup-count"
```

Comment:
should this be `restore-volume-batch-count` ?

Reply:
Will fix it.